apps/spruce/src/pages/waterfall/styles.ts

```diff
@@ -34,7 +35,8 @@ export const Row = styled.div`
   margin-bottom: ${size.s};
 `;
 
-export const InactiveVersion = styled.div`
-  width: ${INACTIVE_WIDTH}px;
+export const InactiveVersion = styled.div<{ hasError: boolean }>`
+  width: ${({ hasError }) =>
+    hasError ? INACTIVE_WITH_ERROR_WIDTH : INACTIVE_WITHOUT_ERROR_WIDTH}px;
```

Comment:
As-is on my laptop this element shrinks to 132px
<img width="254" alt="image" src="https://github.com/user-attachments/assets/ed76c1e4-3487-4b6d-8d3e-91984678c4dd">


I believe we can prevent shrinkage like so. And even 150px is enough vs 160.
```suggestion
  flex-shrink: 0;
  flex-basis: ${({ hasError }) =>
    hasError ? INACTIVE_WITH_ERROR_WIDTH : INACTIVE_WITHOUT_ERROR_WIDTH}px;
```